nginx/tests/test_nginx.py

```diff
@@ -51,10 +68,6 @@ def nginx():
             attempts += 1
             time.sleep(1)
 
-    yield
-
-    subprocess.check_call(args + ["down"], env=env)
```

Comment:
Was this removed by accident?

Reply:
Nope, this function is now only responsible for starting the service, let me add a docstring